File: ClasesTienda/src/Modelos/Pedidos.java

package Modelos;

/**
 *
 * @author jdavi
 */
public class Pedidos {
    private int IdPedido;
    private int IdFactura;
    private int IdUsuario;
    private float MontoAPagar;
    private String Fecha;
    private int IdEstado;
    
    public Pedidos(int IdPedido, int IdFactura, int IdUsuario, String Fecha ){
        this.IdPedido=IdPedido;
        this.IdFactura=IdUsuario;
        this.IdUsuario=IdUsuario;
        this.Fecha=Fecha;
    }
    
    public Pedidos(int IdPedido, int IdFactura, int IdUsuario, float MontoAPagar, String Fecha, int IdEstado ){
        this.IdPedido=IdPedido;
        this.IdFactura=IdUsuario;
        this.IdUsuario=IdUsuario;
        this.MontoAPagar=MontoAPagar;
        this.Fecha=Fecha;
        this.IdEstado=IdEstado;
    }

    public int getIdPedido() {
        return IdPedido;
    }

    public void setIdPedido(int IdPedido) {
        this.IdPedido = IdPedido;
    }

    public int getIdFactura() {
        return IdFactura;
    }

    public void setIdFactura(int IdFactura) {
        this.IdFactura = IdFactura;
    }

    public int getIdUsuario() {
        return IdUsuario;
    }

    public void setIdUsuario(int IdUsuario) {
        this.IdUsuario = IdUsuario;
    }

    public float getMontoAPagar() {
        return MontoAPagar;
    }

    public void setMontoAPagar(float MontoAPagar) {
        this.MontoAPagar = MontoAPagar;
    }

    public String getFecha() {
        return Fecha;
    }

    public void setFecha(String Fecha) {
        this.Fecha = Fecha;
    }

    public int getIdEstado() {
        return IdEstado;
    }

    public void setIdEstado(int IdEstado) {
        this.IdEstado = IdEstado;
    }
}

File: Modelos/ClasesObj/Pedidos.php
<?php

class Pedidos {
    private $IdPedido;
    private $IdFactura;
    private $IdUsuario;
    private $MontoAPagar;
    private $Fecha;
    private $Direccion;
    private $IdEstado;

    public function __construct($IdPedido=null,$IdFactura=null,$IdUsuario=null,$MontoAPagar=null,$Fecha=null,$Direccion=null,$IdEstado=1){
        $this->IdPedido = $IdPedido;
        $this->IdFactura = $IdFactura;
        $this->IdUsuario = $IdUsuario;
        $this->MontoAPagar = $MontoAPagar;
        $this->Fecha = $Fecha;
        $this->Direccion = $Direccion;
        $this->IdEstado = $IdEstado;
    }

    public function SetIdPedido($IdPedido){
        $this->IdPedido = $IdPedido;
    }

    public function GetIdPedido(){
        return $this->IdPedido;
    }

    public function SetIdFactura($IdFactura){
        $this->IdFactura = $IdFactura;
    }

    public function GetIdFactura(){
        return $this->IdFactura;
    }

    public function SetIdUsuario($IdUsuario){
        $this->IdUsuario = $IdUsuario;
    }

    public function GetIdUsuario(){
        return $this->IdUsuario;
    }

    public function SetMontoAPagar($MontoAPagar){
        $this->MontoAPagar = $MontoAPagar;
    }

    public function GetMontoAPagar(){
        return $this->MontoAPagar;
    }

    public function SetFecha($Fecha){
        $this->Fecha = $Fecha;
    }

    public function GetFecha(){
        return $this->Fecha;
    }

    public function SetDireccion($Direccion){
        $this->Direccion = $Direccion;
    }

    public function GetDireccion(){
        return $this->Direccion;
    }

    public function SetIdEstado($IdEstado){
        $this->IdEstado = $IdEstado;
    }

    public function GetIdEstado(){
        return $this->IdEstado;
    }

}


?>



